src/app/features/profile/store/profile.actions.ts

```diff
@@ -1,5 +1,13 @@
-import { createAction } from '@ngrx/store';
-
-const initProfile = createAction('[Profile] Init');
+import { createAction, props } from '@ngrx/store';
+import { UserProfile } from '@interfaces';
 
+const initProfile = createAction('[Profile] Init Profile');
 export const profileActions = { initProfile };
```

Comment:
why if consists of only this one action and doesn't include loadProfile and loadProfileSuccess?